sae_lens/sae.py

```diff
@@ -514,18 +430,7 @@ def encode_jumprelu(
         """
         Calculate SAE features from inputs
         """
-
-        # move x to correct dtype
-        x = x.to(self.dtype)
-
-        # handle hook z reshaping if needed.
-        x = self.reshape_fn_in(x)  # type: ignore
-
-        # handle run time activation normalization if needed
-        x = self.run_time_activation_norm_fn_in(x)
-
-        # apply b_dec_to_input if using that method.
-        sae_in = self.hook_sae_input(x - (self.b_dec * self.cfg.apply_b_dec_to_input))
```

Comment:
we were calling `hook_sae_input()` here after subtracting `b_dec`, but in other `encode()` variants we call `hook_sae_input()` after `reshape_fn_in()` instead. I assumed this different was a bug and not intentional.

Reply:
yep.